docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,298 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport.
+---
+
+<Admonition type="note" title="Important">
+Access to AWS DynamoDB is provided by [**Teleport Application Access**](../../application-access/introduction.mdx) for AWS Console and API.
+</Admonition>
+
+This guide will help you to:
+- Install Teleport Application Service.
+- Set up Teleport to access AWS Console and API.
+- Connect to your DynamoDB through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![DynamoDB Self-Hosted](../../../img/database-access/guides/dynamodb_selfhosted.png)
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+![DynamoDB Cloud](../../../img/database-access/guides/dynamodb_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- A running Teleport cluster, either self hosted or in Teleport Cloud.
+- AWS account with DynamoDB databases.
+- IAM permissions to create IAM roles.
+- A host, e.g., an EC2 instance, where you will run the Teleport Application
+  Service.
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow one of our [getting started guides](../getting-started.mdx) or try our Teleport Application Access [interactive learning track](https://play.instruqt.com/teleport/invite/rgvuva4gzkon).
+</Admonition>
+
+We will assume your Teleport cluster is accessible at `teleport.example.com` and `*.teleport.example.com`. You can substitute the address of your Teleport Proxy Service. (For Teleport Cloud customers, this will be similar to `mytenant.teleport.sh`.)
+
+<Admonition type="note" title="Application Access and DNS">
+(!docs/pages/includes/dns-app-access.mdx!)
+</Admonition>
+
+## Step 1/5. Create an IAM role for DynamoDB access
+
+The first step is to create an IAM role with desired IAM permissions that you wish to grant to a Teleport user. In this example, you will create an IAM role with full DynamoDB access. **Skip this step** if you already have the roles you want to provide access to.
```

Comment:
Which IAM permissions are required for a Teleport user here? Are there any guidelines we can offer?

Reply:
Added one sentence that mentions some AWS-managed policies and copied a note from the original guide. Not sure if this is clearer. Or let me any suggestions. Thanks.